src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -341,6 +342,12 @@ def get_beacon_config(
         max_cpu=bn_max_cpu,
         min_memory=bn_min_mem,
         max_memory=bn_max_mem,
+        labels={
```

Comment:
lets move this into shared_utils so that we don't have to track the `etheruem-package-client` and other strings everywhere

```py
def label_maker(client, type, image, connected_client):
   ... define method here
```

question - can etheruem-package-connected-client be the label instead of specifying connected-el / connected-cl client

Reply:
Good idea.